src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -203,7 +203,7 @@ public boolean accept(Path path) {
             }
             setHadoopBAMConfigurationProperties(filePath, referencePath);
             return SAMHeaderReader.readSAMHeaderFrom(path, ctx.hadoopConfiguration());
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
IllegalArgumentException. I've updated this to make it more specific. Thanks for taking a look!